lib/marked.js

```diff
@@ -1258,18 +1258,19 @@ Parser.prototype.tok = function() {
     case 'list_item_start': {
       body = '';
       var loose = this.token.loose;
+      var checked = this.token.checked;
+      var task = this.token.task;
 
       if (this.token.task) {
-        body += this.renderer.checkbox(this.token.checked);
+        body += this.renderer.checkbox(checked);
       }
 
       while (this.next().type !== 'list_item_end') {
         body += !loose && this.token.type === 'text'
           ? this.parseText()
           : this.tok();
       }
-
-      return this.renderer.listitem(body);
+      return this.renderer.listitem(body, task);
```

Comment:
Should we also pass checked?

```suggestion
      return this.renderer.listitem(body, task, checked);
```